tests/ProductTest.php

```diff
@@ -1674,6 +1675,29 @@ public function testSalesFrequencyIsProperlyExported(): void
         $this->assertSame(1337, $productMock->getField('sales_frequency'));
     }
 
+    public function testUrlIsBuiltFromFirstVariationIfMainVariationIsInactive(): void
```

Comment:
What happens when first variation is inactive as well?

Reply:
`testUrlIsBuiltFromFirstActiveVariation...`?

Reply:
> What happens when first variation is inactive as well?

The product won't be exported, since it has no variations.

Reply:
When main and first variation is inactive and second variation is active, it still gets the right id?

Reply:
> When main and first variation is inactive and second variation is active, it still gets the right id?

Yes of course
![image](https://user-images.githubusercontent.com/20555112/99792237-19b5dd00-2b27-11eb-8084-06aa79d80051.png)
![image](https://user-images.githubusercontent.com/20555112/99792207-0efb4800-2b27-11eb-87ab-8f0c2ff92e1e.png)
![image](https://user-images.githubusercontent.com/20555112/99792167-fdb23b80-2b26-11eb-82fe-75f5b060e47d.png)

Reply:
Then please rename test to "testUrlIsBuiltFromFirst**Active**VariationIfMainVariationIsInactive" so it's clear what the test is about.